generators/release/index.js

```diff
@@ -66,6 +69,9 @@ module.exports = class extends Generator {
          this.applicationName = util.reconcileValue(cmdLnInput.options.applicationName, answers.applicationName, ``);
          this.dockerRegistryId = util.reconcileValue(cmdLnInput.options.dockerRegistryId, answers.dockerRegistryId, ``);
          this.dockerRegistryPassword = util.reconcileValue(cmdLnInput.options.dockerRegistryPassword, answers.dockerRegistryPassword, ``);
+         this.serviceEndpoint = util.reconcileValue(cmdLnInput.options.serviceEndpoint, ``, ``);
```

Comment:
The only way the serviceEndpoint is given is in the k8s sub generator creating it and passing it down to the release sub generator. It will never be “prompted” for, so it wouldn’t be received in answers